<launch>
	<arg name="boxes_topic"					default="/darknet_ros/bounding_boxes"/>
	<arg name="srcimg_topic"				default="/ardrone/front/image_rect_color"/>
	<arg name="threat_image_topic"	default="/threats/potential_images"/>


	<arg name="protoFile"			default="$(find usma_threat_ros)/cfg/coco/pose_deploy_linevec.prototxt"/>
	<arg name="weightsFile"		default="$(find usma_threat_ros)/cfg/coco/pose_iter_440000.caffemodel"/>
	
	<!-- Start the node to evaluate bounding boxes   -->
	<node pkg="usma_threat_ros" type="skeleton_detector.py" name="skeleton_detector" output="screen" >
		<param name="threat_images"		value="$(arg threat_image_topic)" />
		<param name="protoFile"				value="$(arg protoFile)" />
		<param name="weightsFile"			value="$(arg weightsFile)" />
	</node>

	<include file="$(find stop)/launch/stop.launch">
		<arg name="launcher"	value="skeleton_stopper" />
	</include>


</launch>
